ska_conda/ska_builder.py

```diff
@@ -59,30 +81,20 @@ def _build_package(self, name):
         print("Building package %s." % name)
         pkg_path = os.path.join(pkg_defs_path, name)
         cmd_list = ["conda", "build", pkg_path, "--croot",
-                    self.ska_build_dir, "--no-test",
-                    "--no-anaconda-upload"]
+                    self.build_dir, "--no-test",
+                    "--no-anaconda-upload", "--skip-existing"]
         subprocess.run(cmd_list)
 
     def build_one_package(self, name):
         repo = self._get_repo(name)
-        if repo is not None:
-            repo.remote().pull()
```

Comment:
I cut the pulls and the pull version checks below in favor of that --skip-existing option.

Reply:
Yes, pull was never needed as long as you do a fetch and subsequent checkout at the desired tag.